tests/TestCase/Event/EventManagerTest.php

```diff
@@ -145,16 +145,16 @@ public function testOn(): void
     public function testOff(): void
     {
         $manager = new EventManager();
-        $manager->on('fake.event', ['AClass', 'aMethod']);
-        $manager->on('another.event', ['AClass', 'anotherMethod']);
+        $manager->on('fake.event', 'strlen');
+        $manager->on('another.event', 'strlen'(...));
         $manager->on('another.event', ['priority' => 1], 'substr');
 
-        $manager->off('fake.event', ['AClass', 'aMethod']);
+        $manager->off('fake.event', 'strlen'(...));
         $this->assertEquals([], $manager->listeners('fake.event'));
 
-        $manager->off('another.event', ['AClass', 'anotherMethod']);
+        $manager->off('another.event', 'strlen');
```

Comment:
This just shows that a callable function name matches the closure created with first-class callable syntax in the `on()` call.

Reply:
Would we nice if we could use a `(cast)` format for that.